applications/tari_base_node/src/grpc/base_node_grpc_server.rs

```diff
@@ -716,7 +716,8 @@ impl tari_rpc::base_node_server::BaseNode for BaseNodeGrpcServer {
         let block = Block::try_from(request)
             .map_err(|e| Status::invalid_argument(format!("Failed to convert arguments. Invalid block: {:?}", e)))?;
         let block_height = block.header.height;
-        debug!(
+        debug!(target: LOG_TARGET, "Miner submitted block: {}", block);
```

Comment:
```suggestion
        trace!(target: LOG_TARGET, "Miner submitted block: {}", block);
```
This will be a very large log